lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -172,7 +172,8 @@ class FullPageScreenshot extends Gatherer {
           };
         }
 
-        const observedDeviceMetrics = await driver.evaluate(getObservedDeviceMetrics, {
+        const executionContext = driver.executionContext;
```

Comment:
pull out of driver earlier in the fn , so the first usage can use it too?

or maybe better, format the second usage in a way that line limit doesn't happen?

Reply:
sure I'll pull out